crates/commons/src/position.rs

```diff
@@ -0,0 +1,970 @@
+use crate::order_matching_fee_taker;
+use crate::TradeParams;
+use anyhow::bail;
+use anyhow::Context;
+use anyhow::Result;
+use bitcoin::Amount;
+use rust_decimal::prelude::FromPrimitive;
+use rust_decimal::prelude::Signed;
+use rust_decimal::prelude::ToPrimitive;
+use rust_decimal::Decimal;
+use secp256k1::PublicKey;
+use serde::Deserialize;
+use serde::Serialize;
+use time::OffsetDateTime;
+use trade::bitmex_client::Quote;
+use trade::cfd::calculate_margin;
+use trade::cfd::calculate_pnl;
+use trade::ContractSymbol;
+use trade::Direction;
+
+pub type ContractId = [u8; 32];
+
+#[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub enum PositionState {
```

Comment:
🤔 I think this is a bad idea, because we are just exposing the internal state of the coordinator to the maker. Inevitably, we will have to iterate on this model (we already know this work is coming) and this will break the maker. Breaking changes across repositories are particularly annoying to deal with.

I don't think the maker should need all this context to make decisions. I would definitely introduce a dedicated model for the new websocket API.

Reply:
What in particular is a bad idea, moving it into `commons` ? 
Eitherway, it was not necessary after all and I've dropped this commit.

Reply:
I thibk sharing an internal position model with the maker is a bad idea. Glad to hear it is not necessary though.